packages/metal-quartz-icon/src/MetalQuartzIcon.soy

```diff
@@ -1,17 +1,57 @@
 {namespace MetalQuartzIcon}
 
 /**
- * This renders an icon element based on the lexicon icon definition
- * found at http://liferay.github.io/lexicon/content/icons-lexicon/
+ * This renders an icon element based on the Quartz Icon, Quartz Font Awesome
+ * Icon, and Glyphicon definition.
+ * http://getbootstrap.com/components/#glyphicons
  *
  * @param? elementClasses
  * @param spritemap
  * @param symbol
  */
 {template .render}
 	{if isNonnull($symbol) and isNonnull($spritemap)}
-		<svg aria-hidden="true" class="lexicon-icon lexicon-icon-{$symbol}{$elementClasses ? ' ' + $elementClasses : ''}">
-			<use xlink:href="{$spritemap}#{$symbol}" />
-		</svg>
+		{let $quartzIconElementClasses kind="text"}
+			{$elementClasses ? ' ' + $elementClasses : ''}
+		{/let}
+
+		{let $quartzIconVariant kind="text"}
+			{$spritemap == 'quartz-fa' ? 'quartzFa' : $spritemap}
+		{/let}
```

Comment:
Hey @jbalsas @Robert-Frampton, what do you guys think about doing away with the extra param, glyphicons, and font awesome and just allow the user to pass in class names through elementClasses. This way we can use any icon library that uses a span tag for the markup.